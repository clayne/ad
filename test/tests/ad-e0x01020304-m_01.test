ad | -c always -e0x01020304 -m | endian.bin | 0
