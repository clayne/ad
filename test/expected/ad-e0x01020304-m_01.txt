[32m[K0000000000000080[m[K[36m[K:[m[K [41;1m[K0403[m[K [41;1m[K0201[m[K [41;1m[K[m[KFFFF FFFF  FFFF FFFF FF[41;1m[K04[m[K [41;1m[K0302[m[K  [41;1m[K....[m[K.........[41;1m[K...[m[K
[32m[K0000000000000090[m[K[36m[K:[m[K [41;1m[K01[m[KFF FFFF FFFF FFFF  FFFF FFFF FFFF [41;1m[K0403[m[K  [41;1m[K.[m[K.............[41;1m[K..[m[K
[32m[K00000000000000A0[m[K[36m[K:[m[K [41;1m[K0201[m[K [41;1m[K[m[KFFFF FFFF FFFF  FFFF FFFF FFFF FF[41;1m[K04[m[K  [41;1m[K..[m[K.............[41;1m[K.[m[K
[32m[K00000000000000B0[m[K[36m[K:[m[K [41;1m[K0302[m[K [41;1m[K01[m[KFF FFFF FFFF  FFFF FFFF FFFF FFFF  [41;1m[K...[m[K.............
[32m[K0000000000000140[m[K[36m[K:[m[K 0807 0605 [41;1m[K0403[m[K [41;1m[K0201[m[K  [41;1m[K[m[KFF08 0706 05[41;1m[K04[m[K [41;1m[K0302[m[K  ....[41;1m[K....[m[K.....[41;1m[K...[m[K
[32m[K0000000000000150[m[K[36m[K:[m[K [41;1m[K01[m[KFF FFFF FFFF FFFF  FFFF 0807 0605 [41;1m[K0403[m[K  [41;1m[K.[m[K.............[41;1m[K..[m[K
[32m[K0000000000000160[m[K[36m[K:[m[K [41;1m[K0201[m[K [41;1m[K[m[KFFFF FFFF FFFF  FFFF FF08 0706 05[41;1m[K04[m[K  [41;1m[K..[m[K.............[41;1m[K.[m[K
[32m[K0000000000000170[m[K[36m[K:[m[K [41;1m[K0302[m[K [41;1m[K01[m[KFF FFFF FFFF  FFFF FFFF 0807 0605  [41;1m[K...[m[K.............
[32m[K0000000000000180[m[K[36m[K:[m[K [41;1m[K0403[m[K [41;1m[K0201[m[K [41;1m[K[m[KFFFF FFFF  FFFF FFFF FF08 0706  [41;1m[K....[m[K............
[32m[K0000000000000190[m[K[36m[K:[m[K 05[41;1m[K04[m[K [41;1m[K0302[m[K [41;1m[K01[m[KFF FFFF  FFFF FFFF FFFF 0807  .[41;1m[K....[m[K...........
[32m[K00000000000001A0[m[K[36m[K:[m[K 0605 [41;1m[K0403[m[K [41;1m[K0201[m[K [41;1m[K[m[KFFFF  FFFF FFFF FFFF FF08  ..[41;1m[K....[m[K..........
[32m[K00000000000001B0[m[K[36m[K:[m[K 0706 05[41;1m[K04[m[K [41;1m[K0302[m[K [41;1m[K01[m[KFF  FFFF FFFF FFFF FFFF  ...[41;1m[K....[m[K.........
